doc/source/ray-observability/monitoring-debugging/gotchas.rst

```diff
@@ -112,8 +114,12 @@ of Ray Tasks itself, e.g.
   tuner.fit()
 
 This will error with message:
-ValueError: Cannot schedule create_task_that_uses_resources.<locals>.sample_task with the placement group
-because the resource request {'CPU': 10} cannot fit into any bundles for the placement group, [{'CPU': 1.0}].
+
+.. testoutput::
+  :options: +SKIP
+
+    ValueError: Cannot schedule create_task_that_uses_resources.<locals>.sample_task with the placement group
+    because the resource request {'CPU': 10} cannot fit into any bundles for the placement group, [{'CPU': 1.0}].
```

Comment:
Turns out it's possible to test exceptions. Would it be possible to test this `ValueError`?

```
.. testcode::

    assert False

.. testoutput::
    :options: +ELLIPSIS

    Traceback (most recent call last):
      ...
    AssertionError
```

Reply:
Not for this case. The error message is in `error.txt` generated by `tune` not in stdout.